microservices/uyuni-java-parent/pom.xml

```diff
@@ -28,6 +28,35 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <modules>
+        <module>../uyuni-java-common</module>
+        <module>../coco-attestation</module>
+    </modules>
+
+    <!-- Profiles based on the jdk Maven is running on.
+       ~ to ensure the compilation happens with the proper release target
+      -->
+    <profiles>
+        <profile>
+            <id>java-11</id>
+            <activation>
+                <jdk>[11,17)</jdk>
```

Comment:
Are the brackets correct?

Reply:
Yes, that maven way to define version ranges (see https://maven.apache.org/guides/introduction/introduction-to-profiles.html#implicit-profile-activation). This basically defines that for JDK 11 and up to 17 excluded we use the release = 11 flag, otherwise we use release = 17

Reply:
Ok, funny :-)